pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -48,6 +68,7 @@ func NewDynamicFileMapStore(cfg config.Config) (*DynamicFileMapStore, error) {
 	ms := DynamicFileMapStore{}
 	ms.filename = cfg.DynamicFilePath
 	ms.userIDStrict = cfg.DynamicFileUserIDStrict
+	ms.skip = true
```

Comment:
what does `skip` mean? Could we use a name that tells what is being skipped?

Reply:
skip here means we ignore the first time the metric is emitted to solve the authenticator/instance restart issue. Updated the name to be more descriptive.